docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py

```diff
@@ -2,52 +2,126 @@
 import os
 import las
 import requests
+import json
 
 
-def post_feedback(las_client: las.Client, document_id: str, dataset_id: str, verified: dict):
-    logging.info(f'Posting feedback to dataset {dataset_id}: {verified}...')
+def post_feedback_v1(las_client: las.Client, document_id: str, dataset_id: str, verified: dict):
+    document = las_client.get_document(document_id=document_id)
+    old_ground_truth = {g['label']: g['value'] for g in document.get('groundTruth', [])}
 
-    try:
-        document = las_client.get_document(document_id=document_id)
-        old_ground_truth = {g['label']: g['value'] for g in document.get('groundTruth', [])}
-
-        new_ground_truth = {**old_ground_truth, **verified}
-
-        ground_truth = []
-        for label, value in new_ground_truth.items():
-            if isinstance(value, list):
-                if not value:
-                    continue  # Do not write completely empty lines to GT
-                if not isinstance(value[0], list):
-                    value = [[{'label': k, 'value': v} for k, v in line_pred.items()] for line_pred in value]
-            ground_truth.append({
+    new_ground_truth = {**old_ground_truth, **verified}
+
+    ground_truth = []
+    for label, value in new_ground_truth.items():
+        if isinstance(value, list):
+            if not value:
+                continue  # Do not write completely empty lines to GT
+            if not isinstance(value[0], list):
+                value = [[{'label': k, 'value': v} for k, v in line_pred.items()] for line_pred in value]
+        ground_truth.append({
+            'label': label,
+            'value': value,
+        })
+
+    las_client.update_document(
+        document_id=document_id,
+        ground_truth=ground_truth,
+        dataset_id=dataset_id
+    )
+
+
+def post_feedback_v2(las_client: las.Client, document_id: str, dataset_id: str, feedback: dict):
+    def should_post_feedback(item, label):
+        return not item[label].get('automated') or feedback[label].get('isEdited')
+
+    ground_truth = []
```

Comment:
Ground truth for line items need to have a nested structure similar to predictions V1 format